app/page.tsx

```diff
@@ -1,18 +1,22 @@
 "use client";
 
-import { FC } from "react";
+import { FC, useState, useEffect } from "react";
 import { generateKeys } from "./helpers/generateKeys";
 import { Keypair } from "stellar-sdk";
-import { useState } from "react";
-
-interface IKeyPair {
-  publicKey: string;
-  secretKey: string;
-}
+import { IKeyPair } from "./interfaces/keys";
+import InfoModal from "./components/InfoModal";
 
 const Index: FC = () => {
   const [keys, setKeys] = useState({} as IKeyPair);
 
+  useEffect(() => {
+    const init = async () => {
+      const { Modal, Ripple, initTE } = await import("tw-elements");
```

Comment:
Why are we importing this `tw-elements` here?

Reply:
This is recommended in the documentation to avoid the error "document is not defined".
It seems that I chose a too recent library.